drivers/gles3/shaders/scene.glsl

```diff
@@ -1515,6 +1515,12 @@ void main() {
 #CODE : FRAGMENT
 	}
 
+#ifdef DEPTH_USED
+	vec3 ndc = vec3(screen_uv * 2.0 - 1.0, gl_FragDepth);
+	vec4 view_pos = scene_data.inv_projection_matrix * vec4(ndc, 1.0);
```

Comment:
```suggestion
	vec4 view_pos = inv_projection_matrix * vec4(ndc, 1.0);
```

Should use this directly so it properly handles multiview

https://github.com/godotengine/godot/blob/4a0160241fd0c1e874e297f6b08676cf0761e5e8/drivers/gles3/shaders/scene.glsl#L1422